main.tf

```diff
@@ -89,8 +89,12 @@ module "aws_managed_prometheus" {
   count  = var.create_eks && var.enable_amazon_prometheus ? 1 : 0
   source = "./modules/aws-managed-prometheus"
 
-  amazon_prometheus_workspace_alias = var.amazon_prometheus_workspace_alias
-  eks_cluster_id                    = module.aws_eks.cluster_id
+  amazon_prometheus_workspace_alias          = var.amazon_prometheus_workspace_alias
```

Comment:
@vara-bonthu should we consider moving the prometheus module to the example dir with this pr, rather than adding additional properties?

Reply:
Sure, I will move to emr on eks example